dimod/binary/binary_quadratic_model.py

```diff
@@ -1232,7 +1265,36 @@ def from_numpy_matrix(cls, mat, variable_order=None, offset=0,
     @classmethod
     def from_numpy_vectors(cls, linear, quadratic, offset, vartype, *,
                            variable_order=None, dtype=np.float64):
-        """Create a binary quadratic model from NumPy vectors."""
+        """Create a binary quadratic model from NumPy vectors.
+
+        Args:
+            linear (NumPy vector): Linear biases.
```

Comment:
Types should be added to the annotation, not the docstring.

Reply:
Changed definition to use type annotations.